.github/workflows/ci.yml

```diff
@@ -52,14 +52,14 @@ jobs:
         uses: actions/cache@v3
         with:
           path: ${{ env.SCIE_PANTS_DEV_CACHE }}
-          key: ${{ runner.os }}-${{ runner.arch }}-scie-pants-v2
+          key: ${{ matrix.os }}-scie-pants-v4
```

Comment:
So, this is interesting. runner.os / runner.arch is the same for both our macOS runners - which is bad when used for a cache key (See "Configuration: macOS x64" towards the top for both):
+ https://github.com/organizations/pantsbuild/settings/actions/runners/818
+ https://github.com/organizations/pantsbuild/settings/actions/runners/819

@benjyw had you encountered this / known about it?

Reply:
Ugh. I'm guessing this is due to the fact that there used to be no native arm64 GHA runner binary, so we run the x86 one under rosetta.  But it looks like there is a native runner now, so I will switch to that.

Reply:
Ah, OK. That makes total sense then. Thanks.